src/compiler/factory.ts

```diff
@@ -2214,21 +2210,113 @@ namespace ts {
         return tag;
     }
 
-    /* @internal */
     export function createJSDocComment(comment?: string | undefined, tags?: NodeArray<JSDocTag> | undefined) {
         const node = createSynthesizedNode(SyntaxKind.JSDocComment) as JSDoc;
         node.comment = comment;
         node.tags = tags;
         return node;
     }
 
-    /* @internal */
     function createJSDocTag<T extends JSDocTag>(kind: T["kind"], tagName: string): T {
         const node = createSynthesizedNode(kind) as T;
         node.tagName = createIdentifier(tagName);
         return node;
     }
 
+    export function createJSDocUnknownTag() {
+        return createSynthesizedNode(SyntaxKind.JSDocTag) as JSDocUnknownTag;

```

Comment:
is this function even necessary? `createJSDocTag` should be used instead

Reply:
JSDocUnknownTag haven't gov the tagName

Reply:
it has, it extends JSDocTag and therefore contains `tagName` and `comment`